pymc3/distributions/logprob.py

```diff
@@ -283,63 +296,64 @@ def indices_from_subtensor(idx_list, indices):
     )
 
 
-@_logp.register(IncSubtensor)
-@_logp.register(AdvancedIncSubtensor)
-@_logp.register(AdvancedIncSubtensor1)
-def incsubtensor_logp(op, var, rvs_to_values, indexed_rv_var, rv_values, *indices, **kwargs):
+# @_logp.register(IncSubtensor)
+# @_logp.register(AdvancedIncSubtensor)
+# @_logp.register(AdvancedIncSubtensor1)
+# def incsubtensor_logp(op, var, rvs_to_values, indexed_rv_var, rv_values, *indices, **kwargs):
```

Comment:
These needed to be removed (temporarily) since Mixture will being a `Subtensor` will be dispatched here.  We need to take care of these cases i.e when the `Subtensor`s aren't `Mixture`s they should fall back to here. Is there a way we could do that ?